common.go

```diff
@@ -434,3 +441,16 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 
 	return
 }
+
+// GetLanguagesByGitattributes returns a length 1 slice with the language assigned in .gitattributes if the regular expresion
+// matchs with the filename. It is comply with the signature to be a Strategy type.
```

Comment:
This should be called `GetLanguagesByGitAttributes` with an uppercase `A`.  The same for all git attributes functions and variables, except for the name of the file (`.gitattributes`) itself, of course.